src/main/java/com/urswolfer/gerrit/client/rest/http/changes/RevisionApiRestClient.java

```diff
@@ -160,6 +165,13 @@ public MergeableInfo mergeable() throws RestApiException {
         return comments("drafts");
     }
 
+    @Override
+    public Set<String> reviewed() throws RestApiException {
+        String request = getRequestPath() + "/files?reviewed";
```

Comment:
It's probably my fault but I can not find this function in the documentation of Gerrit ([here](https://gerrit-review.googlesource.com/Documentation/). 
If you have found it, could you please link it to me? Thanks

Reply:
Sure here is the link, I was thinking of including the link in the file, but everything else didn't have it either so I left it out.
https://gerrit-review.googlesource.com/Documentation/rest-api-changes.html#list-files

Reply:
Thank you!
Yes, the link is never included in the file so it's better to leave it out for consistency.